src/Endpoint/IdentifierEndpoint.php

```diff
@@ -24,6 +24,8 @@ class IdentifierEndpoint extends EndpointBase implements TokenAwareInterface
     use TokenAwareTrait;
     use ValidationAwareTrait;
 
+    const ENDPOINT_URL_PATH = '/v2/requests/requestIdentifierByCorrelationId/:correlationId';
```

Comment:
At first, I added an abstract method that returned the path. But then I realized that Drupal mocks should also access this path. So I moved the code to the constant. This seems clearer than creating a public method for testing purposes.